src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstance.cs

```diff
@@ -300,11 +300,16 @@ private void CloseRemotingService(bool allowInProcCalls)
         private void StartRemoteIntegrationService(DTE dte)
         {
             // We use DTE over RPC to start the integration service. All other DTE calls should happen in the host process.
-
             if (dte.Commands.Item(WellKnownCommandNames.Test_IntegrationTestService_Start).IsAvailable)
             {
                 dte.ExecuteCommand(WellKnownCommandNames.Test_IntegrationTestService_Start);
             }
+
+            if (AsyncCompletionCondition.Instance.ShouldSkip
```

Comment:
>ShouldSkip [](start = 50, length = 10)

Just for my education, I think this is the key defined in integration test setup to switch between old and legacy completions. Correct?

Reply:
Yes, this is controlled by the environment variable.